src/Compilers/Core/Portable/Collections/HashSetExtensions.cs

```diff
@@ -28,5 +29,18 @@ internal static bool InitializeAndAdd<T>([NotNullIfNotNull(parameterName: nameof
 
             return hashSet.Add(item);
         }
+
+        internal static bool Any<T>(this HashSet<T> hashSet, Func<T, bool> predicate)
```

Comment:
Can we include in the comment why this was introduced so future developers don't delete it?